boot/kernel_os_test.go

```diff
@@ -45,36 +44,19 @@ type: kernel
 vendor: Someone
 `
 
-// coreBootSetSuite tests the abstract bootloader behaviour including
-// bootenv setting, error handling etc., for a core BootSet.
-type coreBootSetSuite struct {
-	baseBootSetSuite
-
-	bootloader *bootloadertest.MockBootloader
-}
-
-var _ = Suite(&coreBootSetSuite{})
-
-func (s *coreBootSetSuite) SetUpTest(c *C) {
-	s.baseBootSetSuite.SetUpTest(c)
-
-	s.bootloader = bootloadertest.Mock("mock", c.MkDir())
-	s.forceBootloader(s.bootloader)
-}
-
-func (s *coreBootSetSuite) TestExtractKernelAssetsError(c *C) {
+func (s *bootenvSuite) TestExtractKernelAssetsError(c *C) {
```

Comment:
should we have a different suite for these tests still? currently we are sharing one suite across multiple files, which I'm not sure if that's something we do already or not